public/pages/Rollups/containers/Rollups/Rollups.tsx

```diff
@@ -0,0 +1,588 @@
+  };
+
+  onActionButtonClick = (): void => {
+    this.setState({ isPopoverOpen: !this.state.isPopoverOpen });
+  };
+
+  closePopover = (): void => {
+    this.setState({ isPopoverOpen: false });
+  };
+
+  closeDeleteModal = (): void => {
+    this.setState({ isDeleteModalVisible: false });
+  };
+
+  showDeleteModal = (): void => {
+    this.setState({ isDeleteModalVisible: true });
+  };
+
+  concatName = (): string => {
+    const { selectedItems } = this.state;
+    let result = "";
+    var item;
+    for (item of selectedItems) {
+      if (selectedItems.indexOf(item) == 0) result = result + item._id;
+      else result = result + ", " + item._id;
+    }
+    return result;
+  };
+  render() {
+    const {
+      totalRollups,
+      from,
+      size,
+      search,
+      sortField,
+      sortDirection,
+      selectedItems,
+      rollups,
+      loadingRollups,
+      isPopoverOpen,
+      isDeleteModalVisible,
+    } = this.state;
+
+    const filterIsApplied = !!search;
+    const page = Math.floor(from / size);
+    const pageCount = Math.ceil(totalRollups / size) || 1;
+
+    const pagination: Pagination = {
+      pageIndex: page,
+      pageSize: size,
+      pageSizeOptions: DEFAULT_PAGE_SIZE_OPTIONS,
+      totalItemCount: totalRollups,
+    };
+
+    const sorting: EuiTableSortingType<RollupItem> = {
+      sort: {
+        direction: sortDirection,
+        field: sortField,
+      },
+    };
+
+    const actionButton = (
+      <EuiButton
+        iconType="arrowDown"
+        iconSide="right"
+        disabled={!selectedItems.length}
+        onClick={this.onActionButtonClick}
+        data-test-subj="actionButton"
+      >
+        Actions
+      </EuiButton>
+    );
+
+    const selection: EuiTableSelectionType<RollupItem> = {
+      onSelectionChange: this.onSelectionChange,
+    };
+
+    const actionItems = [
+      <EuiContextMenuItem
+        key="Edit"
+        icon="empty"
+        disabled={selectedItems.length != 1}
+        data-test-subj="editButton"
+        onClick={() => {
+          this.closePopover();
+          this.onClickEdit();
+        }}
+      >
+        Edit
+      </EuiContextMenuItem>,
+      <EuiContextMenuItem
+        key="Delete"
+        icon="empty"
+        disabled={selectedItems.length == 0}
+        data-test-subj="deleteButton"
+        onClick={() => {
+          this.closePopover();
+          this.showDeleteModal();
+        }}
+      >
+        <EuiTextColor color="danger">Delete</EuiTextColor>
+      </EuiContextMenuItem>,
+    ];
+
+    const rollupsColumns: EuiTableFieldDataColumnType<RollupItem>[] = [
+      {
+        field: "_id",
+        name: "Name",
+        sortable: true,
+        textOnly: true,
+        truncateText: true,
+        render: (_id) => (
+          <EuiLink
+            href={"opendistro_index_management_kibana#/rollup-details?" + _id}
+            onClick={() => this.props.history.push(`${ROUTES.ROLLUP_DETAILS}?id=${_id}`)}
```

Comment:
href is missing the id paramater `?id=${id}`
You can add that and delete onClick, don't need both

Reply:
Added `?id=${id}` to href and removed `onClick`